docs/components/multiselect.md

```diff
@@ -521,6 +533,7 @@ You can pass any input attribute and it will get properly bound to the element.
 
 - `item-template` - The template for each item in the dropdown list
 - `empty` - Slot for the empty state in the dropdown list
+- `dropdown-footer-text` - Slot for footer text in the bottom of the dropdown
```

Comment:
I don't think it makes sense to also allow slotting the dropdown text. This is just meant to be a display string.

Reply:
Slot is useful in case users want to have some control over text style (maybe make something bold, maybe even add a link?)

Reply:
We don't have a use-case for this; I think it should just be a text string as outlined by Design. You could add a CSS variable to control the font weight, size, etc.; however, this is just meant to be informational help text so we want to restrict what can be placed in the footer.

Reply:
But this implementation covers the use case we have plus provides the flexibility should users want to use it differently. Why reduce the functionality if it doesn't introduce any technical complexity or maintenance burden?